File: project/jimmer-sql/src/test/java/org/babyfish/jimmer/sql/dto/DtoBundleTest.java
package org.babyfish.jimmer.sql.dto;

import org.babyfish.jimmer.sql.model.dto.BundledBookView;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class DtoBundleTest {

    @Test
    public void testBundledDtoIsGenerated() {
        Assertions.assertEquals("BundledBookView", BundledBookView.class.getSimpleName());
    }
}

File: project/jimmer-sql-kotlin/src/test/kotlin/org/babyfish/jimmer/sql/kt/dto/DtoBundleTest.kt
package org.babyfish.jimmer.sql.kt.dto

import org.babyfish.jimmer.sql.kt.model.classic.book.dto.BundledBookView
import org.junit.Test
import kotlin.test.assertEquals

class DtoBundleTest {

    @Test
    fun testBundledDtoIsGenerated() {
        assertEquals("BundledBookView", BundledBookView::class.simpleName)
    }
}
